### PR TITLE
#160244936 Implement designs for creating an article

### PR DESCRIPTION
**Pull request by petermwash**
#### What does this PR do?
Implement designs for creating an article.

#### Description of tasks to be completed
Implement designs for creating a new article and it should match the mockup design.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout -b ft-implement-designs-for-creating-an-article-160244936
```
```sh
$ cd ah-titans-frontend
```

> Install the dependencies with the command below.

```sh
$ npm install
```
>Then run the app with the command below.
```sh
$ npm start
```
Now navigate to the URL "http://localhost:8080/article" and should see a page as the one shown on the screenshot below.

screen shot 2018-09-04 at 13 24 45

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160244936](https://www.pivotaltracker.com/story/show/160244936)

### CODE REVIEW

**Comment by dcDalin** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  a `console.log` was left in file (dangerfile.js)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :exclamation: Big PR
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by johnmusiu — APPROVED** [review]

**Review by ronaldndirangu — CHANGES_REQUESTED** [review]
Great work guys. However, I would recommend using a reach text editor like draft.js to improve the user experience.

**Comment by petermwash** [comment]
@ronaldndirangu I think if we use draft.js it will change the whole view of the page and it will differ from the mockup we had earlier. I would recommend we consult with the PO on that and probably create a story for this.

**Review by ronaldndirangu — APPROVED** [review]

**Comment by ronaldndirangu** [comment]
@petermwash that makes sense. We can make the changes later if the team requires it.

**Review by migot01 — APPROVED** [review]

**Comment by jmnyarega** [comment]
Please update your branch @petermwash

**Comment by janetnim** [comment]
Also, you should also consider adding upload image option on the template

File: src/Header/components/index.js [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
           <li><Link to="/login">Login</Link></li>
           <li><Link to="/signup">Signup</Link></li>
-        </div>
-      </ul>
-    </nav>
-  </header>
+        </ul>
+      </Col>
+    </Col>
+
```
Comment (jmnyarega) [inline src/Header/components/index.js]: Lose this line

**Comment by petermwash** [comment]
@jmnyarega Fixed.

**Pull request merged**